README.md

```
@@ -29,6 +29,15 @@ Biome
 ```shell
 pn biome check --write
 ```
+sync astro
+```shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)